### PR TITLE
2362 Laos EOC Visuals

### PR DESCRIPTION
**Pull request by IgorNadj**
### Changes:
- https://github.com/beyondessential/tupaia/pull/2217
- https://github.com/beyondessential/tupaia/pull/2221

### CODE REVIEW

File: packages/database/src/migrations/20210309020032-MalariaMapOverlaysLaosEOC-modifies-data.js [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+const malariaCaseMapOverlayBySubDistrict = {
+  ...malariaBaseMapOverlayBySubDistrict,
+  id: 'LAOS_EOC_Total_Malaria_Cases_By_Sub_District',
+  name: 'Malaria Cases by Sub District',
+  dataElementCode: 'Total_Positive_Malaria_Cases',
+};
+const malariaDeathMapOverlayBySubDistrict = {
+  ...malariaBaseMapOverlayBySubDistrict,
+  id: 'LAOS_EOC_Total_Malaria_Deaths_By_Sub_District',
+  name: '	Malaria Deaths by Sub District',
```
Comment (PShoe) [inline packages/database/src/migrations/20210309020032-MalariaMapOverlaysLaosEOC-modifies-data.js:55]: 'Malaria Deaths by District' 

as per https://github.com/beyondessential/tupaia-backlog/issues/2376#issuecomment-800839365 
cc- @billli0

File: packages/database/src/migrations/20210309020032-MalariaMapOverlaysLaosEOC-modifies-data.js [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+    },
+    periodGranularity: 'one_day_at_a_time',
+  },
+  countryCodes: '{"LA"}',
+  projectCodes: '{"laos_eoc"}',
+};
+const malariaCaseMapOverlayBySubDistrict = {
+  ...malariaBaseMapOverlayBySubDistrict,
+  id: 'LAOS_EOC_Total_Malaria_Cases_By_Sub_District',
+  name: 'Malaria Cases by Sub District',
```
Comment (PShoe) [inline packages/database/src/migrations/20210309020032-MalariaMapOverlaysLaosEOC-modifies-data.js:49]: 'Malaria Deaths by District' 

as per https://github.com/beyondessential/tupaia-backlog/issues/2376#issuecomment-800839365 
cc- @billli0

File: packages/database/src/migrations/20210315001650-AddMapOverlayMeaslesDeathCasesLaosEOC-modifies-data.js [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+    },
+    periodGranularity: 'one_day_at_a_time',
+  },
+  countryCodes: '{"LA"}',
+  projectCodes: '{"laos_eoc"}',
+};
+const measlesCaseMapOverlayBySubDistrict = {
+  ...measlesBaseMapOverlayBySubDistrict,
+  id: 'LAOS_EOC_Total_Measles_Cases_By_Sub_District',
+  name: 'Measles Cases by Sub District',
```
Comment (PShoe) [inline packages/database/src/migrations/20210315001650-AddMapOverlayMeaslesDeathCasesLaosEOC-modifies-data.js:49]: 'Malaria Cases by District' 

as per https://github.com/beyondessential/tupaia-backlog/issues/2376#issuecomment-800839365 
cc- @billli0

File: packages/database/src/migrations/20210315001650-AddMapOverlayMeaslesDeathCasesLaosEOC-modifies-data.js [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+  dataElementCode: 'Total_Positive_Measles_Cases',
+};
+const measlesDeathMapOverlayBySubDistrict = {
+  ...measlesBaseMapOverlayBySubDistrict,
+  id: 'LAOS_EOC_Total_Measles_Deaths_By_Sub_District',
+  name: '	Measles Deaths by Sub District',
```
Comment (PShoe) [inline packages/database/src/migrations/20210315001650-AddMapOverlayMeaslesDeathCasesLaosEOC-modifies-data.js:55]: 'Measles Deaths by District' 

as per https://github.com/beyondessential/tupaia-backlog/issues/2376#issuecomment-800839365 
cc- @billli0

**Review by chris-bes — APPROVED** [review]

**Pull request merged**